### ISSUE TITLE
bsPopover can't handle line breaks

### ISSUE THREAD

**Issue by moorepants** [ebailey78/shinyBS#47]
This fails to render in my app:

`bsPopover(content='abc\n')`

whereas this does:

`bsPopover(content='abc')`

**Comment by Persedes** [comment]
add a  < br > (minus the spaces...) instead of \n in your string

**Comment by jefferys** [comment]
First, thanks for this great package and for your work to get it up and running again after major breaking changes upstream from you.

Unfortunately, I need to expand the scope of this issue. I too ran into the problem mentioned above, but it is slightly more complex than just not rendering "\n". Not a major problem once I figured out why things weren't working, but I wasted some time getting things working.

It seems popovers and tooltips will fail silently with actual line breaks in the string `content` in the **source code**, even when not rendered. This is hard to show here, but for example, if my source code looks something like:

```
    bsPopover( 'id', title='Title', content="<p>Paragraph 1</p><p>Paragraph 2</p>" )
```

it works, but if it looks like

```
    bsPopover( 'if', title='Title', content="<p>Paragraph 1</p>
                                             <p>Paragraph 2</p>" )
```

with any real line break inside the quoted string, nothing will show. It also might be that all following popovers will fail to show after one breaks due to this. I am using shinyBS 0.61 and working on a mac in Rstudio, in case that matters.

Thanks again!

**Comment by blosloos** [comment]
Yes, thanks for this great package! I just would like to add to the above comment that I observed the same behaviour, on R 4.1.2, shinyBS version 0.61, Windows. With coding line breaks in `content=""` nothing is shown ...
